applications/ShapeOptimizationApplication/custom_python/shape_optimization_python_application.cpp

```diff
@@ -56,12 +56,30 @@ using namespace pybind11;
     KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS(m, NORMALIZED_SURFACE_NORMAL);
 
     // Optimization variables
-    KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS(m, OBJECTIVE_SENSITIVITY);
-    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, OBJECTIVE_SURFACE_SENSITIVITY);
-    KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS(m, MAPPED_OBJECTIVE_SENSITIVITY);
-    KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS(m, CONSTRAINT_SENSITIVITY);
-    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, CONSTRAINT_SURFACE_SENSITIVITY);
-    KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS(m, MAPPED_CONSTRAINT_SENSITIVITY);
+    KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS(m, DF1DX);
```

Comment:
I dont understand this, maybe a more descriptive name would be better
Just suggesting

Reply:
I agree. F stands for objective function and C for constraint function. But having a collection of optimization results and looking at variables like "dObjectiveFunctiondX" or "dConstraint1dX" is not probably even worse. Instead, knowing that there is an objective and some constraints. The implemented variables look a bit more readable imo.